api/proto/relay/relay.proto

```diff
@@ -34,7 +34,7 @@ message GetChunksRequest {
 
   // If this is an authenticated request, this should hold the ID of the requester. If this
   // is an unauthenticated request, this field should be empty.
-  uint64 requester_id = 2;
+  bytes requester_id = 2;
```

Comment:
From the code below, this is interpreted as operator ID. It'll be helpful to document it here so the users know how to  set it.

Reply:
I've renamed this field to `operator_id`. Is that sufficient, or do you think I should add additional documentation?